test/mkdirp.coffee

```diff
@@ -0,0 +1,30 @@
+command = require '../lib/coffee-script/command'
+path = require 'path'
+fs = require 'fs'
+
+test "mkdirp", ->
+  deepDir = 'test/mkdirp/deep/dir'
+  process.argv = [
+    'node', 'coffee'
+    '-o', deepDir
+    '-c', __filename
+  ]
+  command.run()
+
+  compiledPath = path.join(
+    deepDir
+    path.basename(__filename, '.coffee') + '.js'
+  )
+
+  # Coffee's test suit doesn't support async task.
+  # Here some hack.
+  setTimeout ->
+    ok fs.existsSync(compiledPath)
```

Comment:
Just move the opening paren before the arrow and we don't have to do this gross line continuation.